pkgs/servers/home-assistant/custom-components/cync_lights/default.nix

```diff
@@ -0,0 +1,28 @@
+{
+  lib,
+  fetchFromGitHub,
+  buildHomeAssistantComponent,
+}:
+
+buildHomeAssistantComponent rec {
+  owner = "nikshriv";
+  domain = "cync_lights";
+  version = "1.0.1";
+
+  src = fetchFromGitHub {
+    inherit owner;
+    repo = "cync_lights";
+    rev = "f4923bf2ab343a679a78495f66289ab9961f43ba";
+    hash = "sha256-MLMHpPhcICzQQ6nT9joViwhsp+x9E0kwVrlEmV8XJ0s=";
+  };
+
+  dependencies = [ ];
+
+  meta = with lib; {
+    changelog = "https://github.com/nikshriv/cync_lights/commits/main/";
+    description = "Home Assistant Integration for controlling Cync switches, plugs, and bulbs";
+    homepage = "https://github.com/nikshriv/cync_lights";
+    maintainers = with maintainers; [ ndarwincorn ];
+    license = licenses.unlicensed;
```

Comment:
That is not a valid license, and upstream has no license, so copyright likely applies. Please clarify the situation with upstream.

Reply:
https://github.com/nikshriv/cync_lights/issues/106

Will update once the repo is licensed.